src/app/dashboard-widgets/environment-widget/environment-widget.module.ts

```diff
@@ -5,11 +5,13 @@ import { RouterModule } from '@angular/router';
 
 import { MomentModule } from 'angular2-moment';
 
+// import { FeatureFlagModule } from '../../feature-flag/feature-flag.module';
+
 import { DeploymentsModule } from '../../space/create/deployments/deployments.module';
 import { EnvironmentWidgetComponent } from './environment-widget.component';
 
 @NgModule({
-  imports: [CommonModule, DeploymentsModule, FormsModule, RouterModule, MomentModule],
+  imports: [CommonModule, DeploymentsModule, FormsModule, RouterModule, MomentModule/*, FeatureFlagModule */],
```

Comment:
What's the purpose of this code that is commented out? And above? If not needed I'd prefer it removed.

Reply:
I think it is commented out so that you can turn it on when needed. But I would agree, we should be able to leave that on and use it when needed, since we will continue to need to use it.

Reply:
this link to the wiki page example: https://github.com/fabric8-ui/fabric8-ui/wiki/fabric8-toggles#my-feature-is-a-component-i-want-to-hideshow
but could be remove. even better if we find concrete use-cases.